pkg/storage/tsdb/config.go

```diff
@@ -98,6 +98,14 @@ const (
 	// partitioner aggregates together two bucket GET object requests.
 	DefaultPartitionerMaxGapSize = uint64(512 * 1024)
 
+	// NewBlockDiscoveryDelayMultiplier is the factor used (multiplied with BucketStoreConfig.SyncInterval) to determine
+	// when querying a newly uploaded block is required.
+	// For example, if BucketStoreConfig.SyncInterval is 15 minutes, then a querier will require that it can query all
```

Comment:
(nit)
```suggestion
	// For example, if BucketStoreConfig.SyncInterval is 15 minutes and NewBlockDiscoveryDelayMultiplier is 3, then a querier will require that it can query all
```

Reply:
I'm not sure this adds much - the constant is defined to be 3 a few lines later.